src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/AvoidUninstantiatedInternalClasses.cs

```diff
@@ -31,7 +31,7 @@ public sealed class AvoidUninstantiatedInternalClassesAnalyzer : DiagnosticAnaly
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Performance,
                                                                              DiagnosticHelpers.DefaultDiagnosticSeverity,
-                                                                             isEnabledByDefault: DiagnosticHelpers.EnabledByDefaultForVsixAndNuget,
```

Comment:
This has to be disabled because the fixer is essentially a no-op. See http://index/?query=AvoidUninstantiatedInternalClassesFixer&rightProject=Microsoft.CodeQuality.Analyzers&file=Maintainability%5CAvoidUninstantiatedInternalClasses.Fixer.cs&line=26.

@jinujoseph @kuhlenh, we need to update our enabled by default list.